docs/spec/consensus/light-client.md

```diff
@@ -1,113 +1,145 @@
-# Light Client
+# Lite Client
 
-A light client is a process that connects to the Tendermint Full Node(s) and then tries to verify the Merkle proofs
-about the blockchain application. In this document we describe mechanisms that ensures that the Tendermint light client
+A lite client is a process that connects to the Tendermint Full Node(s) and then tries to verify application data using the Merkle proofs. In this document we describe mechanisms that ensures that the Tendermint lite client
 has the same level of security as Full Node processes (without being itself a Full Node).
 
-To be able to validate a Merkle proof, a light client needs to validate the blockchain header that contains the root app hash.
-Validating a blockchain header in Tendermint consists in verifying that the header is committed (signed) by >2/3 of the
-voting power of the corresponding validator set. As the validator set is a dynamic set (it is changing), one of the
-core functionality of the light client is updating the current validator set, that is then used to verify the
-blockchain header, and further the corresponding Merkle proofs.
+## Lite client requirements from Tendermint and Proof of Stake modules
+
+Before explaining lite client mechanisms and operations we need to define some requirements expected from the Tendermint blockchain. Tendermint provides a deterministic, Byzantine fault-tolerant, source of time (called
```

Comment:
So I don't think that BFT has a huge amount to with lite clients.

Weak Subjectivity is entirely in terms of clients local clock. 

It is slightly relevant in an IBC context where BFT time is the local clock because the lite client is another blockchain.

Reply:
I think we need to constrain drifts between local time and BFT time to be able to come up with some guarantees. Otherwise (at least in theory) clocks can drift arbitrarily.

Reply:
I thought we're going to remove BFT time, no? https://github.com/tendermint/tendermint/issues/2840

Reply:
The plan isn't to remove BFT time but to make it proposer based so the attack is 2f+1 instead of f+1 to accelerate time.